fdbserver/storageserver.actor.cpp

```diff
@@ -4550,7 +4550,7 @@ ACTOR Future<Void> reportStorageServerState(StorageServer* self) {
 
 ACTOR Future<Void> storageServerCore(StorageServer* self,
                                      StorageServerInterface ssi,
-                                     MockLogSystem* mockLogSystem = nullptr) {
+                                     std::shared_ptr<MockLogSystem> mockLogSystem = std::shared_ptr<MockLogSystem>()) {
```

Comment:
nit: I would still write this as `= nullptr`. Yes, it is the case that a default initialized `shared_ptr` has to be equal to `nullptr`, but I still like being explicit.